### ISSUE TITLE
mAP is Zero for 2 Classes while Training DetectNet for Multi-class Object Detection

### ISSUE THREAD

**Issue by adithya-p** [NVIDIA/DIGITS#2000]
I am training DetectNet on COCO Dataset for persons and vehicle classes and there are no dontcare conditions. I have downloaded entire COCO dataset and converted COCO annotations to KITTI labels using python code.

custom classes: dontcare, person, vehicle
solver type: ADAM
subtract mean: none
learning rate: 2.5e-07
epochs: 500
policy: Exponential Decay
gamma: 0.99
I got the network from here: [detectnet_network-2classes.prototx](https://github.com/NVIDIA/caffe/blob/caffe-0.15/examples/kitti/detectnet_network-2classes.prototxt) 
I replaced **1248*384** with **640*640** in the network, giving a pre-trained-model:  [bvlc_googlenet.caffemodel](http://dl.caffe.berkeleyvision.org/bvlc_googlenet.caffemodel)

My mAP is zero even after 100 epochs. I have checked similar issues and responses from @lukeyeager @gheinrich @dusty-nv @jbarker-nvidia and tried them, but they didn't solve my issue. Can you give a detailed insight on how to fix the issue?

**Comment by mahxn0** [comment]
I also have this problem , i extract car lable from coco data and then convert to kitti label . after 74epoch , mAP is ZERO!!!   if you solve the problem can share some experience?

**Comment by jbarker-nvidia** [comment]
@adithya-p @mahxn0 I'm afraid you haven't really provided enough information to diagnose why your training might be failing.

As @adithya-p mentioned there are many similar issues that have been opened related to failure of DetectNet to converge, it's worth checking those carefully for hints.  One other thing you might want to check is whether the coverage map (visualized in DIGITS during inference) is non-zero anywhere in a test image.  That will tell you whether the problem is related to the final thresholding and bounding box generation or whether the network is failing to learn sensitivity to the target object classes at all.
